### ISSUE TITLE
Picture converter

### ISSUE THREAD

**Issue by mccoy321** [TheLastMillennial/HDPictureViewerConverter2#4]
The picture converter file will not open so i can not convert any pictures. Is there a way past it or a different file?

**Comment by TheLastMillennial** [comment]
The HDPictureViewerConverter.exe file won't run? Did you extract everything from the zip folder first?

What OS are you using (Windows 10?) do you get any error messages?

In the meantime you can use the third party converter: https://taricorp.gitlab.io/hdpictureconverter-rs/

**Comment by mccoy321** [comment]
Yes, I extracted everything from the zip file but it just wouldn't open . I have a MacBook air.  i am using the online converter and it works great. So its okay
Thank you.
________________________________

**Comment by TheLastMillennial** [comment]
Mac isn't officially supported but it should work if you use Wine. Glad the other converter works!
